### DISCUSSION TITLE
Prompt suggestions language to change based on selected language

### DISCUSSION THREAD

**Discussion by pascal-beaulne** [open-webui/open-webui#3913] [General]
**Is your feature request related to a problem? Please describe.**
The prompt suggestions' title and content in a new chat does not change based on the language.

**Describe the solution you'd like**
I would like the prompt suggestions' title and content to adapt automatically based on the selected language. This would ensure that users receive contextually accurate suggestions based on the language of their choice.

**Comment by s3r1msultan** [comment]
Have you found any solution to fix it?

**Reply by rgaricano** [reply]
there a pull proposed about: https://github.com/open-webui/open-webui/pull/11208
